### PR TITLE
Use cache for profile images that do not require hot reload, Rename alt tags for logo image

### PR DESCRIPTION
**Pull request by rawrrencee**
# Changelog:

remove getTime() from images where no hot reload is possible, rename alt tags for logo images

hot reload is only triggered when uploading a new profile pic, which is not triggered on any of these pages. therefore, it is ok to use the cached version of the image and not load the same image multiple times

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

### CODE REVIEW

File: src/layouts/Auth/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const AuthLayout = ({
           >
             <div className={style.logoContainer}>
               <a href="/" className={style.logo}>
-                <img src="/resources/images/logo.svg" width="32" className="mr-2" alt="Clean UI" />
+                <img src="/resources/images/logo.svg" width="32" className="mr-2" alt="Digi Dojo" />
                 <div className={style.name}>{logo}</div>
```
Comment (natalieloke) [inline src/layouts/Auth/index.js]: smol comment - we can use the DIGI DOJO const enum defined in the earlier PR haha

Comment (rawrrencee) [reply src/layouts/Auth/index.js]: ok!

**Review by natalieloke — APPROVED** [review]
lgtm thanks lawrence

**Review by y1bs — APPROVED** [review]
lgtm!

**Pull request merged**